w09h03/test/pgdp/filter/VideoContainerTests.java

```diff
@@ -0,0 +1,74 @@
+package pgdp.filter;
+
+import org.bytedeco.javacv.FrameRecorder;
+import org.junit.jupiter.api.Test;
+
+import java.io.FileNotFoundException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class VideoContainerTests {
```

Comment:
```suggestion
public class VideoContainerTest {
```

Test files should end with `Test` and not `Test`

Reply:
I guess you mean `Tests`...

In my opinion we should allow `*Test(s)` and update the contribution guidelines accordingly since sometimes it just makes more sense. That's what gets suggested by most IDEs as well.